birthdays-project/BirthdayApp/views.py

```diff
@@ -0,0 +1,21 @@
+from django.shortcuts import render
+from django.http import Http404
+
+from .models import User
+
+
+def index(request):
+    userDB = User.objects.all()
```

Comment:
I like the all_users, or queryset suggestion. I changed it to employee_queryset as that helps me best learn and remember what I'm working with.